seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/schema/SeaTunnelSchema.java

```diff
@@ -42,7 +44,8 @@
 import java.util.Map;
 
 public class SeaTunnelSchema implements Serializable {
-    public static final String SCHEMA = "schema";
+
+    public static final Option<Schema> SCHEMA = Options.key("schema").objectType(Schema.class).noDefaultValue().withDescription("SeaTunnel Schema");
```

Comment:
SeaTunnelSchema implements Serializable but `Option<Schema> SCHEMA` don't implements Serializable.

Reply:
It doesn't need Serializable because static.